### DISCUSSION TITLE
Rendering of index pages looks off and is fully aligned in center

### DISCUSSION THREAD

**Discussion by Skarlso** [thuliteio/doks#1231] [How-to guides] [answered]
Hello.

I migrated our website to the latest version from below `1.0.0`.

Everything is super cool except for a single thing where the `_index.md` pages are rendered super funky even though all pages are rendered correctly, the index pages are aligned in the middle and the TOC is gone from the side.

This was not the case previously. The index pages rendered like regular pages, nicely the same way. Here is an example:

Current view:
<img width="1905" alt="Screenshot 2024-04-29 at 14 09 48" src="https://github.com/gethyas/doks/assets/182850/31e597f9-6510-4015-8507-36d13ea2bc69">

Old view:
<img width="1919" alt="Screenshot 2024-04-29 at 14 12 28" src="https://github.com/gethyas/doks/assets/182850/56417676-3993-4849-9ced-41ee894c5bc4">

Any ideas how to bring back this topic view / index rendering / whatever that is? :D

**Comment by h-enk** [comment]
That's hard to tell this way — could you share your repo?

**Comment by Skarlso** [comment]
Sure, no problem. Here is the website: https://ocm.software.

And here is the repository: https://github.com/open-component-model/ocm-website/

That particular page comes from https://github.com/open-component-model/ocm-website/tree/main/content/docs/cli-reference/add

**Comment by h-enk** [comment, marked as answer]
You'll need to make some customizations to make the docs list template behave like the docs single template.

First, copy `node_modules/@hyas/doks-core/layouts/_default/baseof.html` to `layouts/_default/baseof.html` and update the logic to display the container (remove `(.IsNode)`) — new contents:

```html
<!doctype html>
<html lang="{{ .Site.LanguageCode | default "en" }}" data-bs-theme="{{ site.Params.doks.colorMode | default "auto" }}">
  {{ partial "head/head" . }}
  {{ partial "head/body-class" . }}
  <body class="{{ delimit (.Scratch.Get "class") " " }}"{{ if eq site.Params.doks.scrollSpy true }} data-bs-spy="scroll" data-bs-target="#toc" data-bs-root-margin="0px 0px -60%" data-bs-smooth-scroll="true" tabindex="0"{{ end }}>
    {{ partial "header/header" . }}
    <div class="wrap container-{{ site.Params.doks.containerBreakpoint | default "lg" }}" role="document">
      <div class="content">
      {{ if and (eq site.Params.doks.containerBreakpoint "fluid") (or (not (in .Site.Params.mainSections .Type))) }}<div class="container p-0">{{ end }}
        {{ block "main" . }}{{ end }}
      {{ if and (eq site.Params.doks.containerBreakpoint "fluid") (or (not (in .Site.Params.mainSections .Type))) }}</div>{{ end }}
      </div>
    </div>
    {{ block "sidebar-prefooter" . }}{{ end }}
    {{ block "sidebar-footer" . }}{{ end }}
    {{ partial "footer/footer" . }}
    {{ partial "footer/script-footer" . }}
    {{ if eq site.Params.doks.toTopButton true -}}
      {{ partial "footer/to-top" . }}
    {{ end }}
  </body>
</html>
```

Next, copy `node_modules/@hyas/doks-core/layouts/_default/single.html` to `layouts/_default/list.html` (making the docs list template like the docs single template)

Lastly, add to `assets/scss/common/_custom.scss`:

```scss
.list.section.docs .content {
    padding-top: 0;
    padding-bottom: 0;
}
```

**Reply by Skarlso** [reply]
Holy moly. :D I'll try to do that in a sec. :) Thank you!

**Reply by Skarlso** [reply]
Oh brother that worked perfectly. Thank you so much! 🙇
